ipython_widgets/static/widgets/js/widget.js

```diff
@@ -443,7 +441,7 @@ define(["nbextensions/widgets/widgets/js/manager",
     widgetmanager.WidgetManager.register_widget_model('WidgetModel', WidgetModel);
 
 
-    var WidgetView = Backbone.View.extend({
+    var WidgetInterface = {
```

Comment:
It is not only about removing jquery but allowing alternate drop-in replacements to Backbone.View like [D3View](https://github.com/akre54/Backbone.D3View) or [NativeView](https://github.com/akre54/Backbone.NativeView). 
I edited the description.